src/WebJobs.Script/Extensions/ScriptLoggingBuilderExtensions.cs

```diff
@@ -14,21 +16,24 @@ public static class ScriptLoggingBuilderExtensions
         internal static readonly string[] AllowedCategoryPrefixes = new[]
         {
             "Microsoft.Azure.WebJobs",
-            "Function",
-            "Worker",
-            "Host"
+            ScriptConstants.LogCategoryFunction,
+            ScriptConstants.LogCategoryWorker,
+            ScriptConstants.LogCategoryHost,
+            ScriptConstants.LogCategoryHostGeneral
         };
 
         public static void AddDefaultWebJobsFilters(this ILoggingBuilder builder)
         {
             builder.SetMinimumLevel(LogLevel.None);
             builder.AddFilter((c, l) => Filter(c, l, LogLevel.Information));
+            builder.AddFilter(ScriptConstants.LogCategoryAlways, LogLevel.Information);
         }
 
         public static void AddDefaultWebJobsFilters<T>(this ILoggingBuilder builder, LogLevel level) where T : ILoggerProvider
         {
             builder.AddFilter<T>(null, LogLevel.None);
-            builder.AddFilter<T>((c, l) => Filter(c, l, LogLevel.Trace));
+            builder.AddFilter<T>((c, l) => Filter(c, l, level));
+            builder.AddFilter<T>(ScriptConstants.LogCategoryAlways, LogLevel.Information);
```

Comment:
cc @anthonychu 

@brettsam  - Introduced new log category to ensure logs are always emitted. Let me know if there is a better way to achieve this.